### DISCUSSION TITLE
Synchronize two sessions

### DISCUSSION THREAD

**Discussion by gabrielbaeg** [frangoteam/FUXA#1392] [General]
Hi!

I was testing scripts with two FUXA pages opened and I discovered something, these two sessions don't synchronize with each other so if I modify something in one window that modification doesn't reflect in the second one. This is a problem because if two users working in the same instance of FUXA they will rewrite each other's project.

**Comment by unocelli** [comment]
Hi, the current FUXA architecture does not provide the concurrency and update management for the editor.

**Reply by lengfeng10086** [reply]
Hello, currently FUXA pro supports adding DAQ database types in the settings. How can it be done and how should it be done

**Reply by unocelli** [reply]
@lengfeng10086 https://www.youtube.com/watch?v=UuE_uLSXWGM&t=16s
